src/Serilog.Sinks.AzureTableStorage/Sinks/AzureTableStorageWithProperties/AzureBatchingTableStorageWithPropertiesSink.cs

```diff
@@ -33,7 +33,8 @@ public class AzureBatchingTableStorageWithPropertiesSink : PeriodicBatchingSink
 		private readonly IFormatProvider _formatProvider;
 		private readonly CloudTable _table;
 		private readonly string _additionalRowKeyPostfix;
-		private const int _maxAzureOperationsPerBatch = 100;
```

Comment:
The whitespace is also a bit weird in this file and the next two in the diff as well ;)